content/blog/2022-02-09-predictkube-scaler.md

```diff
@@ -0,0 +1,163 @@
++++
+title = "Introducing PredictKube - an AI-based predictive autoscaler for KEDA made by Dysnix"
+date = 2022-02-09
+author = "Daniel Yavorovych (Dysnix), Yuriy Khoma (Dysnix), Zbynek Roubalik (KEDA), Tom Kerkhove (KEDA)"
+aliases = [
+"/blog/introducing-predictkube-an-ai-based-predictive-autoscaler-for-keda-made-by-dysnix"
+]
++++
+
+## Introducing PredictKube—an AI-based predictive autoscaler for KEDA made by Dysnix
+
+[Dysnix](https://dysnix.com/) has been working with big high-loaded projects for a long time,
+and the efficient scaling demand is what their team comes across each day.
+The engineers have understood that dealing with traffic fluctuations and preparations of infrastructure by bare hand is
+totally inefficient as you need to deploy more resources _before_ the traffic increases,
+not at the moment the event happens. It is too late to scale when the big traffic wave has already come.
```

Comment:
```suggestion
not at the moment the event happens. This strategy is problematic for two reasons: first, because it's often too late to scale when traffic has already arrived and second, resources will be overprovisioned and idle during the times that traffic isn't present.
```